src/test/java/org/fundacionjala/coding/ana/PersistenceTest.java

```diff
@@ -0,0 +1,68 @@
+package org.fundacionjala.coding.ana;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * number persistence.
+ *
+ * @author Ana Maria Mamani Zenteno.
+ */
+public class PersistenceTest {
+    private Persistence persistence;
+    private static final int VALUE39 = 39;
+    private static final int EXPECTED3 = 3;
+    private static final int VALUE4 = 4;
+    private static final int VALUE999 = 999;
+    private static final int EXPECTED4 = 4;
+    private static final int VALUE0 = 0;
+    private static final int EXPECTED0 = 0;
+    /**
+     * call tha class Persistence.
+     *
+     * @throws Exception after.
+     */
+    @Before
+    public void setUp() throws Exception {
+        persistence = new Persistence();
+    }
+
+    /**
+     * test of cant persistence.
+     */
+    @Test
+    public void testPersistenceCantPersistenceComparato() {
+        int result = persistence.cantPersistence(VALUE39);
+        assertEquals(EXPECTED3, result);
+        System.out.print(result);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)